docs/creating-your-site.md

```diff
@@ -1,16 +1,29 @@
 # Creating your site
 
-After you've [installed] Material for MkDocs, you can bootstrap your project
-documentation using the `mkdocs` executable. Go to the directory where you want
-your project to be located and enter:
+After you've [installed] Material for MkDocs, you can bootstrap your documentation project then preview your documentation and finaly build your site using the `mkdocs` executable.
+
+## Bootstrap your project
+
+Go to the directory where you want your project to be located and enter:
 
 ```
 mkdocs new .
 ```
 
 Alternatively, if you're running Material for MkDocs from within Docker, use:
 
-=== "Unix, Powershell"
+
+=== "Unix"
+
+    ```
+    docker run --rm \
+               --user $(id -u):$(id -g) \
+               --volume $(pwd):/docs \
+               squidfunk/mkdocs-material \
+               new . # bootstrap project
```

Comment:
Please remove the comment, this is not necessary, as it's described in the paragraph above.